pkg/httpmetrics/transport.go

```diff
@@ -45,7 +46,8 @@ var (
 		},
 		[]string{"code", "method", "host", "service_name", "revision_name", "ce_type"},
 	)
-	seenHostMap = make(map[string]int)
+	seenHostMap   = make(map[string]int)
```

Comment:
Can we use sync.Map instead of adding a lock?

Reply:
changed